src/ComponentBase.tsx

```diff
@@ -48,13 +48,16 @@ export class IroComponentBase extends Component<Props, State> {
   }
 
   render(props) {
-
     const eventHandler = this.handleEvent.bind(this);
 
-    const rootProps = {
-      onMouseDown: eventHandler,
-      onTouchStart: eventHandler
+    let rootProps = {
+      onMouseDown: eventHandler
     };
+    if (/(iPhone|iPad|iPod|iOS|Android)/i.test(navigator.userAgent)) {
```

Comment:
Hi!

Please could you explain this? I'm not sure what device or browser you're using where this event being `onTouchStart` vs `ontouchstart` would be an issue. Sniffing the user-agent string is usually considered a bad practice and should only be done if absolutely necessary, so if I can understand the problem more then maybe we can find a better alternative here.

Thanks!

Reply:
Hi,
The Object rootProps containing key called 'onTouchStart' will be used when adding event listener. If you look through the code on binding event, you will see that if current DOM does not has the touchstart event it will set the 'onTouchStart ' event as a customized event which will not be triggered by the browser in PC. But, obviously, the browsers in PC mainly call 'mousedown' event so it is not any problem, while the touchstart can work for mobile devices. However, if the device is a touch screen desktop/laptop, touching event will not be triggered on customized 'TouchStart' event, so the user can't do any drag or move actions to pick a color. If we change onTouchStart to ontouchstart, it works on any devices perfectly.
![image](https://user-images.githubusercontent.com/30002112/91248316-f75e2e80-e786-11ea-95dd-d7b04c49a4cb.png)

Actually, this judgement 'if (/(iPhone|iPad|iPod|iOS|Android)/i.test(navigator.userAgent))' is not needed, the reason why I putting here to distinguish onTouchStart and 'ontouchstart' is because I not sure whether your onTouchStart could be used on other processes.

Reply:
Okay, I did some testing and `ontouchstart` seems to work fine on iOS and Android devices as far as I can tell, so I just pushed a fix to use that instead of `onTouchStart`

Thanks for the help! :)